shared/python/dashboard_info.py

```diff
@@ -76,16 +77,29 @@ def __init__(self, home_dir):
         # better to generate the fields and accessors than have a full file of boilerplate
         # info type, system type, singular, plural
         dashboard_fields = [
-            (InfoType.config,  SystemType.exp,    'config',  'configs'),
-            (InfoType.results, SystemType.exp,    'data',    'data'),
-            (InfoType.results, SystemType.exp,    'status',  'statuses'),
-            (InfoType.results, SystemType.exp,    'graph',   'graphs'),
-            (InfoType.results, SystemType.exp,    'summary', 'summaries'),
-            (InfoType.config,  SystemType.subsys, 'config',  'configs'),
-            (InfoType.results, SystemType.subsys, 'status',  'statuses'),
-            (InfoType.results, SystemType.subsys, 'output',  'output')
+            (InfoType.config,  SystemType.exp,    'config',    'configs'),
+            (InfoType.results, SystemType.exp,    'data',      'data'),
+            (InfoType.results, SystemType.exp,    'status',    'statuses'),
+            (InfoType.results, SystemType.exp,    'graph',     'graphs'),
+            (InfoType.results, SystemType.exp,    'summary',   'summaries'),
+            (InfoType.config,  SystemType.subsys, 'config',    'configs'),
+            (InfoType.results, SystemType.subsys, 'status',    'statuses'),
+            (InfoType.results, SystemType.subsys, 'output',    'output'),
+            (InfoType.results, SystemType.telemetry, 'telemetry', 'telemetries'),
```

Comment:
Is there any reason why telemetry can't just be a subsystem that is under the subsystem header? I don't see the need for adding another category

Reply:
Sorry, I forgot to push those changes.
Submitted the change in the latest commit.